FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -908,14 +997,14 @@ static BaseType_t prvConnectNetwork( NetworkContext_t * pxNetworkContext )
                                                    &xNetworkCredentials,
                                                    mqttexampleTRANSPORT_SEND_RECV_TIMEOUT_MS,
                                                    mqttexampleTRANSPORT_SEND_RECV_TIMEOUT_MS );
-            xConnected = ( xNetworkStatus == TLS_TRANSPORT_SUCCESS );
+            xConnected = ( xNetworkStatus == TLS_TRANSPORT_SUCCESS ) ? true : false;
```

Comment:
Would it be better to use a `BaseType_t`?

Reply:
Might as well. I tried to stick with either `bool` or `BaseType_t` consistently and settled on `bool` since that's what MQTT uses, but since the return type for this function was changed a while back I can just convert the `bool`s here